test/CMakeLists.txt

```diff
@@ -80,4 +80,23 @@ set_test_target_parameters(benchmarks benchmarks.cc)
 add_subdirectory(system)
 if(CELERITY_DETAIL_INTEGRATION_TESTING)
   add_subdirectory(integration)
+endif()
+
+# Optionally link region map tests against Cairo for debug drawing
+
+find_path(CAIRO_INCLUDE_DIRS
+    NAMES cairo.h
+    PATH_SUFFIXES cairo
+)
+
+find_library(CAIRO_LIBRARIES
+    NAMES cairo
+)
+
+target_compile_definitions(region_map_tests PUBLIC -DGUGU=23)
```

Comment:
What? Missing NOCOMMIT?

Reply:
This was a test to see if you are paying attention 😎 No actually I had some difficulties getting compile definitions to work on the test executable (I had to set it on the object file instead). Will remove, lol.